ConsumerApi/appsettings.json

```diff
@@ -14,16 +14,30 @@
                 "MinimumBackoff": 2,
                 "MaximumBackoff": 120
             }
+        },
+        "SqlDatabaseConfiguration": {
+            "EnableHealthCheck": false
```

Comment:
All db health checks should be enabled by default

Reply:
done